src/modules/token/fungible/components/SendForm/SendForm.js

```diff
@@ -89,23 +90,27 @@ const SendForm = (props) => {
     setReference({ target: { value: '' } });
   }, []);
 
-  const isValid = useMemo(
-    () =>
-      [amount, recipient, reference, recipientChain, sendingChain, token].reduce((result, item) => {
+  const isFormValid = useMemo(() => {
+    const isFieldsValid = [amount, recipient, reference, recipientChain, sendingChain].reduce(
+      (result, item) => {
         result =
           result &&
           !item?.error &&
           (!item?.required || item?.value !== '') &&
           !Object.keys(result).length;
 
         return result;
-      }, true),
-    [amount, recipient, reference, recipientChain, sendingChain]
-  );
+      },
+      true
+    );
+    const isTokenValid = !!token && Object.keys(token).length;
+    return isFieldsValid && isTokenValid;
```

Comment:
```suggestion
    return areFieldsValid && isTokenValid;
```

`is` for singular and `are` for plural?